pkg/termination/handler_test.go

```diff
@@ -102,7 +102,7 @@ var _ = Describe("Handler Suite", func() {
 		var testMachine *machinev1.Machine
 
```

Comment:
why is this change needed now?

Reply:
It seems to me that new version on envtest requires pre-created test namespace, otherwise unit tests fail

Reply:
it'd be nice to specify that in the commit desc